opensrp-chw/src/main/res/layout/activity_malaria_profile_chw.xml

```diff
@@ -0,0 +1,160 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It would be good to have all the views used by the module moved to the module repo